src/components/views/context_menus/MessageContextMenu.tsx

```diff
@@ -462,10 +462,6 @@ export default class MessageContextMenu extends React.Component<IProps, IState>
                     }
                     onClick={showPermalink ? this.onCopyPermalinkClick : this.onPermalinkClick}
                     label={showPermalink ? _t('Copy link') : _t('Share')}
-                    element="a"
-                    href={permalink}
-                    target="_blank"
-                    rel="noreferrer noopener"
```

Comment:
these are important for middle click and other anchor semantics

Reply:
I was thinking there might be a way to use generics here but I don't think there would be a way to avoid the duplicity of the `element` prop and the type

Reply:
I'm a bit confused as to how this was working before - did we convert something to TypeScript that sits between here and the AccessibleButton?

Extending all the types is not great - if there's a type out there that determines the attributes based on the tag name then we can use that, otherwise I guess we're left with no choice :(

Reply:
I am not sure what you mean by before - the copy link button is new, this didn't work before.

Yeah, I am afraid our only option is to use all :(

Reply:
the code was there previously, but it's been removed/altered by the PR, so it did work at some point.

It'd be the comment below which made it work, at a glance:
```
// XXX: Typescript signature for AccessibleButton doesn't work properly for non-inputs like `a`
```

Reply:
Oh, I see, should be fixed now! Sorry, for not seeing this earlier :sweat_smile: